__test__/manage-project.test.js

```diff
@@ -0,0 +1,81 @@
+const { By, until, Key } = require('selenium-webdriver-3');
+const { driver } = require('./helper');
+const { load } = require('./Pages');
+
+describe('manage-project', () => {
+  beforeEach(async () => {
+    await load();
+  });
+  it('should create a project and update data for a project.', async () => {
+    const url = await driver.getCurrentUrl();
+    await driver.navigate().to(`${url}/login`);
+    (await driver).sleep(200);
+    //Login Process
+    await driver.findElement(By.id('username')).sendKeys('test-tpo@plant-for-the-planet.org');
+    await driver.findElement(By.name('action')).click();
+    await driver.findElement(By.id('password')).sendKeys('CcCFg2enJ@C7XrV3ukqHbYYbaN-2hBW7hh6_Ye8kBorZAwczZfdM*TJnMLdgpbi');
+    await driver.findElement(By.name('action')).click();
```

Comment:
After password, find name /id code

<img width="966" alt="Screen Shot 2021-03-18 at 1 31 57 PM" src="https://user-images.githubusercontent.com/5401575/111626469-58feec80-87ee-11eb-8fe6-95540b7c3015.png">

Add OTP generated by the app and then find name('action) and click.